src/nuget.js

```diff
@@ -0,0 +1,123 @@
+const findReachableUrls = require('find-reachable-urls');
+const got = require('got');
+const isUrl = require('is-url');
+const { xml2js } = require('xml-js');
+
+const cache = require('./utils/cache');
+const log = require('./utils/log');
+const repositoryUrl = require('./registries/repository-url');
+
+const getLatestVersion = async (pkg) => {
+  let response;
+  try {
+    response = await got.get(`https://api.nuget.org/v3-flatcontainer/${pkg}/index.json`);
+  } catch (err) {
+    if (err.statusCode === 404) {
+      return log('Package not found', pkg);
```

Comment:
Minor, the return value of `log` is `undefined`, but from a readability point of view it is better to put the return in a new line so it's obvious that we return `undefined` here.

Reply:
Yea, it took me a bit to realize that's what was going on with that code after copying it from one of the other plugins. It's fixed here now.